papergres.go

```diff
@@ -688,6 +705,7 @@ func exec(q *Query, nonQuery bool) *Result {
 			if err != nil {
 				return err
 			}
+			meta.LastInsertId, _ = res.LastInsertId()
 			meta.RowsAffected, _ = res.RowsAffected()
```

Comment:
Same as above

Reply:
Same as above, please read the comment regarding the LastInsertID, RowsAffected is implemented in a similar fashion by the `lib/pq` and sqlx package. So, the scope of error is close to zero, since all major database drivers already implement it.